presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -228,6 +228,16 @@ public static Boolean stIsEmpty(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
         return deserialize(input).isEmpty();
     }
 
+    @SqlNullable
```

Comment:
@SqlNullable means that the function can take in nulls and may produce nulls, but current implementation cannot handle nulls. It throws NPE in `geometry.isEmpty()`.

Let's remove @SqlNullable and change return type to `boolean`. This will make it so that input will never be null and therefore doesn't need to be handled explicitly.